admin/app/templates/withoutnav_template.html

```diff
@@ -2,7 +2,7 @@
 
 {% block fullwidth_content %}
   <div id="content">
-    {% if current_service %}
+    {% if current_service and current_user.is_authenticated and current_user.belongs_to_service(current_service.id) %}
```

Comment:
@dalehurwitz I don't think `belongs_to_service` is defined? Searched the codebase, can't find it. Seems to be newly defined here: https://github.com/alphagov/notifications-admin/commit/558ae87baa806842c73db3e638e30c83745f4690 are you able to more or less just pull in the changes in that commit?

Reply:
Apologies, I managed to revert this change somehow before I committed it. Fixed.

Reply:
@dalehurwitz all good!